shell/server/proxy.js

```diff
@@ -975,6 +975,15 @@ const apiTokenForRequest = (req, hostId) => {
   } else if (auth && auth.slice(0, 6).toLowerCase() === "basic " &&
              apiUseBasicAuth(req, hostId)) {
     token = (new Buffer(auth.slice(6).trim(), "base64")).toString().split(":")[1];
+  } else if (isWebsocket && req.url.startsWith("/.sandstorm-well-known/token/")) {
```

Comment:
Request:

Use `/.well-known/sandstorm/token` not `/.sandstorm-well-known`

Rationale: `/.well-known/` is standardized so apps should know strange things might happen there.